varats/varats/data/reports/blame_report.py

```diff
@@ -329,34 +326,6 @@ def shorthand(cls) -> str:
         """Shorthand for this report."""
         return cls.SHORTHAND
```

Comment:
`SHORTHAND` got replaced by `shorthand()`
```suggestion
        return cls.shorthand()
```